airbyte-workers/src/main/resources/entrypoints/check.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
+
```

Comment:
Im not sure if this is intentional or not but there are a bunch of shell scripts without a shebang, please correct me if this is intentional and I can remove